src/style.css

```diff
@@ -141,8 +145,9 @@ li {
 }
 
 .close {
-  position: relative;
-  left: 45%;
+  position: absolute;
+  right: 0;
+  top: 0;
```

Comment:
I suggest adding some space between the edge of the div and the close icon, to make it look slightly more professional. Maybe add `top: 5px` and `right: 5px`